frontend/src/global.scss

```diff
@@ -309,7 +309,7 @@ style files without adding already imported styles. */
 
 // Overlays styles
 #bottom-notice {
-    z-index: 999;
+    z-index: 10500;
```

Comment:
:thinking: suggestion for future: Let's pull out out z-indexes to vars file and order them there. This way conflicts will (hopefully) become more apparent.